src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -542,7 +541,6 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
                     .AddHeader(Alignment.Right, width: 3)
                     .AddHeader()
                     .StartRowDefinition()
-                        .AddPropertyColumn("HandleCount")
```

Comment:
And also in `src\System.Management.Automation\engine\Types_Ps1Xml.generated.cs`, there are references to 'HandleCount' and 'Handles' in both alias property definition and `DefaultDisplayPropertySet`. They should be updated too I think.